sql/core/src/main/scala/org/apache/spark/sql/execution/basicOperators.scala

```diff
@@ -263,6 +265,10 @@ case class Except(left: SparkPlan, right: SparkPlan) extends BinaryNode {
   protected override def doExecute(): RDD[InternalRow] = {
     left.execute().map(_.copy()).subtract(right.execute().map(_.copy()))
   }
+
+  override def outputsUnsafeRows: Boolean = children.forall(_.outputsUnsafeRows)
+  override def canProcessUnsafeRows: Boolean = true
```

Comment:
seems it's already promised in https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/rowFormatConverters.scala#L95-L106